src/coders.ts

```diff
@@ -10,52 +10,37 @@ export function encodeAllocations(allocation: Allocation) {
   );
 }
 
-export function encodeExit(exit: Exit) {
-  return defaultAbiCoder.encode(
-    [
+const exitTy = [
```

Comment:
Nit: I've seen this `-Ty` suffix in use elsewhere... but is `-Type` better?

Reply:
Renamed to `exitABI` as this is what it is really